File: CWH Python Beginner's 10hr Course/Chapters/Ch4/Ch4_PS/problem4.py
# Write a program to sum a list with four number
list = []

a = int(input("Enter Number 1: "))
list.append(a)
b = int(input("Enter Number 2: "))
list.append(b)
c = int(input("Enter Number 3: "))
list.append(c)
d = int(input("Enter Number 4: "))
list.append(d)

print(list[0]+list[1]+list[2]+list[3])
# Alternate and easier way is
print(sum(list))

File: CWH C Language Beginner's 10hr Course/Chapters/Chapter 1/CH1_PS/problem4.c
// Write a program to calculate simple interest using principal, rate of interest, years

#include <stdio.h>

int main(){
    float interest,principal,years,simple_interest;
    printf("Enter the Principal Value: ");
    scanf("%f",&principal);
    printf("Enter the Interest rate in \%: ");
    scanf("%f",&interest);
    printf("Enter the Number of years: ");
    scanf("%f",&years);
    simple_interest = (interest*principal*years)/100;
    printf("The simple interest for the following terms is %f\n", simple_interest);
    return 0;
}